tweet-api/api/inference.py

```diff
@@ -0,0 +1,44 @@
+import csv
+import urllib.request
+
+import numpy as np
+from scipy.special import softmax
+from transformers import AutoModelForSequenceClassification, AutoTokenizer
+
+task='emotion'
+MODEL = f"cardiffnlp/twitter-roberta-base-{task}"
+tokenizer = AutoTokenizer.from_pretrained(MODEL)
+
+# download label mapping
+mapping_link = f"https://raw.githubusercontent.com/cardiffnlp/tweeteval/main/datasets/{task}/mapping.txt"
+with urllib.request.urlopen(mapping_link) as f:
+    html = f.read().decode('utf-8').split("\n")
+    csvreader = csv.reader(html, delimiter='\t')
```

Comment:
Would it be simpler to skip the csvreader here - since it's just 4 lines with `<int>\t<label>`?
(Not super important, but it gives the impression that we need to "use the big guns" - the `csv` package - to do a very simple task!)